internal/captain/values.go

```diff
@@ -242,7 +245,12 @@ func (u *TimeValue) String() string {
 
 func (u *TimeValue) Set(v string) error {
 	if v == "now" {
-		u.Time = ptr.To(time.Now())
+		latest, err := model.FetchLatestRevisionTimeStamp(authentication.LegacyGet())
```

Comment:
I know it's gross to use legacy functions, but we don't have access to prime as far as I can tell.

Reply:
That's not a precendent, that's legacy code that hasn't yet been refactored.

This is kind of a weird situation though. From the perspective of captain.TimeValue it seems presumptuous to equate "now" to a platform API call.

I think what we should do is update TimeValue so that it can communicate what the original input was, and then we update it to the appropriate timestamp in the runner.